assets/schema_input.json

```diff
@@ -10,25 +10,42 @@
             "sample": {
                 "type": "string",
                 "pattern": "^\\S+$",
-                "errorMessage": "Sample name must be provided and cannot contain spaces"
+                "errorMessage": "Sample name must be provided and cannot contain spaces",
+                "meta": ["id"]
             },
             "fastq_1": {
                 "type": "string",
+                "format": "file-path",
+                "exists": true,
                 "pattern": "^\\S+\\.f(ast)?q\\.gz$",
                 "errorMessage": "FastQ file for reads 1 must be provided, cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'"
             },
             "fastq_2": {
+                "type": "string",
+                "format": "file-path",
+                "exists": true,
                 "errorMessage": "FastQ file for reads 2 cannot contain spaces and must have extension '.fq.gz' or '.fastq.gz'",
-                "anyOf": [
-                    {
-                        "type": "string",
-                        "pattern": "^\\S+\\.f(ast)?q\\.gz$"
-                    },
-                    {
-                        "type": "string",
-                        "maxLength": 0
-                    }
-                ]
+                "pattern": "^\\S+\\.f(ast)?q\\.gz$"
+            },
+            "condition": {
+                "type": "string",
+                "pattern": "^\\S+$",
+                "meta": ["condition"]
+            },
+            "reference": {
+                "type": "string",
+                "pattern": "^[ACTGNactgn]+$",
```

Comment:
Double-checked that, thanks! Some tools such as `minimap2` only take into account `N`s as ambiguous bases, thus it is better to avoid other kinds of ambiguous bases.